### PR TITLE
add Lenovo Servers SR530/SR550/X3250M6 and fix SR650

### PR DESCRIPTION
**Pull request by nxzz**
This pull request adds Lenovo Servers SR530/SR550/X3250M6 and fix SR650.

The product datasheet can be found below.
SR530 https://lenovopress.lenovo.com/lp1045.pdf
SR550 https://lenovopress.lenovo.com/lp1046.pdf
SR650 https://lenovopress.lenovo.com/lp1050.pdf
x3250M6 https://lenovopress.lenovo.com/lp0096.pdf

### CODE REVIEW

File: device-types/Lenovo/SR530.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+interfaces:
+  - name: Ethernet 1
+    type: 1000base-t
+  - name: Ethernet 2
+    type: 1000base-t
+  - name: XCC
+    type: 1000base-t
+    mgmt_only: true
+console-ports:
+  - name: Front VGA
```
Comment (danner26) [inline device-types/Lenovo/SR530.yaml:24]: VGA ports are not meant to be modelled this way, lets remove the console-ports/vga adapters

File: device-types/Lenovo/SR550.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+interfaces:
+  - name: Ethernet 1
+    type: 1000base-t
+  - name: Ethernet 2
+    type: 1000base-t
+  - name: XCC
+    type: 1000base-t
+    mgmt_only: true
+console-ports:
+  - name: Rear VGA
```
Comment (danner26) [inline device-types/Lenovo/SR550.yaml:24]: VGA ports are not meant to be modelled this way, lets remove the console-ports/vga adapters

File: device-types/Lenovo/SR650.yaml [hunk 3]
```diff
@@ -14,3 +16,6 @@ power-ports:
     type: iec-60320-c14
   - name: PSU1
     type: iec-60320-c14
+console-ports:
+  - name: Rear VGA
```
Comment (danner26) [inline device-types/Lenovo/SR650.yaml:20]: VGA ports are not meant to be modelled this way, lets remove the console-ports/vga adapters

File: device-types/Lenovo/system_x3250_M6.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    type: iec-60320-c14
+  - name: PSU 2
+    type: iec-60320-c14
+interfaces:
+  - name: Ethernet 1
+    type: 1000base-t
+  - name: Ethernet 2
+    type: 1000base-t
+console-ports:
+  - name: Rear VGA
```
Comment (danner26) [inline device-types/Lenovo/system_x3250_M6.yaml:20]: VGA ports are not meant to be modelled this way, lets remove the console-ports/vga adapters

**Review by danner26 — CHANGES_REQUESTED** [review]
VGA adapters are not console ports, and thus should not be modelled how you currently are. Please remove those from the definitions

File: device-types/Lenovo/system_x3250_M6.yaml (hunk 4 above)
Comment (nxzz) [reply device-types/Lenovo/system_x3250_M6.yaml]: removed

File: device-types/Lenovo/SR650.yaml (hunk 3 above)
Comment (nxzz) [reply device-types/Lenovo/SR650.yaml]: removed

File: device-types/Lenovo/SR550.yaml (hunk 2 above)
Comment (nxzz) [reply device-types/Lenovo/SR550.yaml]: removed

File: device-types/Lenovo/SR530.yaml (hunk 1 above)
Comment (nxzz) [reply device-types/Lenovo/SR530.yaml]: removed

**Review by danner26 — APPROVED** [review]

**Comment by danner26** [comment]
Thank you for your contribution!

**Pull request merged**